src/platform/user/profile/vet360/tests/actions/transactions.unit.spec.js

```diff
@@ -27,92 +25,6 @@ const analyticsSectionName = 'bar';
 describe('validateAddress', () => {
   it('verify return data', () => {
     const dispatch = sinon.spy();
-    const mockReturn = {
```

Comment:
I don't believe this mock return was never actually used because these tests are run in an environment where Vet360/VA Profile is not available so we completely bypass making REST requests,